.github/workflows/publish-to-test-pypi.yml

```
@@ -7,10 +7,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@master
-    - name: Set up Python 3.10
+    - name: Set up Python 3.11
       uses: actions/setup-python@v5
       with:
-        python-version: 3.10
+        python-version: 3.11
```

Based on https://github.com/nrdg/cloudknot/actions/runs/8396694493/job/22998596231: 
```suggestion
        python-version: "3.11"
```